src/UI/images/Image.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import styles from '../../styles.scss';
+import PropTypes from 'prop-types';
+
+function Image({ image, ...props }) {
+  return (
+    <div className={styles.images}>
+      <img src={image} {...props}></img>
```

Comment:
`...props` is typically passed to the wrapper of the component but lets say you want to do it specifically for the image

Reply:
also you're still not using the self-closing tags. + always add alt attributes to your images

Reply:
alt attribute is here
![image](https://user-images.githubusercontent.com/101064758/194744380-bda2b573-252e-4512-9ab2-92d1bf0de0ae.png)

Reply:
Image component can be used anywhere for this reason I add alt attribute in props